main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
```

Comment:
This is a code quality issue. You are reading the entire file into memory which could be a problem if the file is large. Consider using a streaming approach to read the file. This way, you are only loading a small part of the file into memory at any given time. For instance, you could use the `ijson` library to stream the JSON file.